blocks/library/block/index.php

```diff
@@ -13,8 +13,17 @@
  * @return string Rendered HTML of the referenced block.
  */
 function gutenberg_render_block_core_reusable_block( $attributes ) {
-	$reusable_block = get_post( $attributes['ref'] );
-	if ( ! $reusable_block ) {
+	if ( empty( $attributes['ref'] ) ) {
+		return '';
+	}
+
+	$reusable_block_id = absint( $attributes['ref'] );
+	if ( ! $reusable_block_id ) {
+		return '';
+	}
```

Comment:
This check isn't necessary. If `ref` is not an integer then the parser will omit it from `$attributes`.

Reply:
Fixed, thanks for spotting that.